.github/workflows/nightly.yml

```diff
@@ -0,0 +1,53 @@
+name: Nightly release
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - workflow
+
+jobs:
+  nightly:
+
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3.0.2
```

Comment:
Why the full version? Why not v4?

Reply:
why v4?

Reply:
Because its the latest release with a newer NodeJS version and will not lead to depreciation warnings. See https://github.com/actions/checkout/releases/tag/v4.0.0

Reply:
this is false, what it does is make the app not being able to be build, the only way to *fix* it and use the latest version of node is to use --openssl-legacy-provider, which just defaults back to the same provider as being used in v3.0.2